04/script.js

```diff
@@ -0,0 +1,323 @@
+const tasksProm = prompt(`Number
+String
+Boolean
+Comparison
+Ternary
+Prompt
+Confirm (or)
+Confirm (if)
+Default (or)
+Default (if)
+Login
+Currency
+Scissors
+чорний пояс
+`)
+
+if (tasksProm === "Number") {
+     // Number: odd
+     const promptValue = prompt('Введіть число');
+
+     if (isNaN(promptValue)) {
+          alert('Ви ввели не число!')
+     }
+
+     if (promptValue % 2 === 0) {
+          alert('введено парне число')
+     } else {
+          alert('введено рядок або число не парне')
+     }
+}
+
+else if (tasksProm === "String") {
+     // String: lexics
+     const arrayRandomWord = prompt('введіть будь-що')
+          .includes('a') === true
+          ? alert('літеру "a" знайдено') : alert('літеру "a" НЕ знайдено');
+}
+
+else if (tasksProm === "Boolean") {
+     // Boolean + Boolean: if
+
+     const isMale = confirm('Ви чоловік?');
+     const likesPizza = confirm('Вам подобається піца?');
+
+     let genderInfo, pizzaInfo;
+
+     if (isMale) {
+          genderInfo = "Ви чоловік";
+     } else {
+          genderInfo = "Ви жінка";
+     }
+
+     if (likesPizza) {
+          pizzaInfo = "і ви любите піцу";
+     } else {
+          pizzaInfo = "і ви НЕ любите піцу";
+     }
+     const answer = alert(`${genderInfo}, ${pizzaInfo}`);
+}
+
+else if (tasksProm === "Comparison") {
+     // Comparison: sizes
+
+     const sizeUSA = prompt('Введіть ваш розмір(США)');
+
+     let internationalSize;
+
+     if (sizeUSA <= 2) {
+          internationalSize = "XS"
+
+     } else if (sizeUSA > 2 && sizeUSA <= 6) {
+          internationalSize = "S"
+
+     } else if (sizeUSA > 6 && sizeUSA <= 10) {
+          internationalSize = "M"
+
+     } else if (sizeUSA > 10 && sizeUSA <= 14) {
+          internationalSize = "L"
+
+     } else if (sizeUSA > 14 && sizeUSA <= 17) {
+          internationalSize = "XL"
+
+     } else {
+          internationalSize = "XL+"
+     }
+
+     const UAH = alert(`
+Ваш розмір USA: ${sizeUSA}
+Ваш розмір UA: ${Number(sizeUSA) + 38}
+Ваш розмір international: ${internationalSize}
+`)
+}
+
+else if (tasksProm === "Ternary") {
+     // Ternary
+     const male = confirm('Ви чоловік?');
+     male === true ? alert('Ви чоловік') : alert('Ви жінка');
+
+}
+
+else if (tasksProm === "Prompt") {
+
+     // Prompt: or
+     let age = prompt('How old are you?');
+
+     if (age === null || age === undefined || age === '' || isNaN(age)) {
+          alert('Відповідь некоректна');
+     } else {
+          alert(`Вам ${age} роки(ів)`);
+     }
+}
+
+else if (tasksProm === "Confirm (or)") {
+     // Confirm: or this days
+
+     confirm("Шопінг?") || alert("ти бяка");
+}
+
+else if (tasksProm === "Confirm (if)") {
+
+     //Confirm: if this days
+     let shoping = confirm('Шопінг?');
+
+     if (shoping === false) {
+          alert('ти бяка');
+     }
+}
+
+else if (tasksProm === "Default (or)") {
+
+     // Default: or
+     let surname = prompt('Прізвище?') || "Шевченко";
```

Comment:
const 
also in new ES standard was introduced operator ??
and ?? suits more for this task